dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

```diff
@@ -8,5 +8,8 @@ USER root
 # Install curl and bash
 # Install ssh for cloning ssh-repositories
 # Install less for handling git diff properly
+# Install sshpass for handling passwords for SSH keys
 RUN yum install -y sudo git bzip2 which bash curl openssh less && \
+    wget https://netcologne.dl.sourceforge.net/project/sshpass/sshpass/1.06/sshpass-1.06.tar.gz -O sshpass.tar.gz && \
+    tar -xvf sshpass.tar.gz && cd sshpass-1.06 && ./configure && make install && cd .. && rm -rf sshpass-1.06 && \
```

Comment:
we should extract "1.06" as a variable so we don't forget to update it in future.

Reply:
done